docker/Dockerfile-jetson-jp4

```diff
@@ -0,0 +1,64 @@
+# Ultralytics YOLO 🚀, AGPL-3.0 license
+# Builds ultralytics/ultralytics:jetson-jp4 image on DockerHub https://hub.docker.com/r/ultralytics/ultralytics
+# Supports JetPack4.x for YOLOv8 on Jetson Nano, TX2, Xavier NX, AGX Xavier
+
+# Start FROM https://catalog.ngc.nvidia.com/orgs/nvidia/containers/l4t-cuda
+FROM nvcr.io/nvidia/l4t-cuda:10.2.460-runtime
+
+# Set environment variables
+ENV APP_HOME /usr/src/ultralytics
+
+# Downloads to user config dir
+ADD https://github.com/ultralytics/assets/releases/download/v0.0.0/Arial.ttf \
```

Comment:
Can I know why do you install the fonts separately?

Reply:
Good question. This step is done in all our Docker images. Maybe @glenn-jocher has more insights about the reason.

Reply:
@Ahelsamahy @lakshanthad we don't include it in the pip package as it can be large (larger than the package itself), and is not always needed, mostly for training non-ascii character classes, like Chinese or Arabic names. It is downloaded on demand at runtime, but we pre-download it in the Docker image for convenience.